<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ideal Weight</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Favicon.qrc">
    <normaloff>:/Favicon/фавикон.png</normaloff>:/Favicon/фавикон.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Form{
	/*background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:0, y2:1, stop:0 #f06d5b, stop:1 /*#ee5162#ef4f62 rgb(255, 78, 119));*/
	
	/*background-image: url(:/background/222);*/
	
	/*background-image: url(:/background/Снимок1.PNG);*/
	/*background-image: url(:/Снимок1.PNG);*/
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.875, stop:0 #f06d5b, stop:1 rgb(255, 78, 119));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.875, stop:0 #f06d5b, stop:1 rgb(255, 35, 112));
}</string>
  </property>
  <widget class="QSlider" name="horizontalSlider_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider#horizontalSlider_3::handle {
	background-color: rgb(255, 255, 255);
	width: 10px;
	height: 10px;
    border-radius: 5px;

 }
 /*QSlider#horizontalSlider::sub-page:horizontal {
     background: rgb(49, 255, 46);
 }*/</string>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>431</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="footer_img.qrc">:/Bez_Imeni-1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>431</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Simplex_IV25</family>
     <pointsize>15</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
/*font-family: &quot;Montserrat&quot;,Verdana,sans-serif;*/
font: 15pt /*&quot;Segoe UI&quot;*/&quot;Simplex_IV25&quot;;
font-weight: 500;</string>
   </property>
   <property name="text">
    <string>ОБХВАТ ЗАПЯСТЬЯ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>541</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Simplex_IV25</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
/*font-family: &quot;Montserrat&quot;,Verdana,sans-serif;*/
/*font: 27pt &quot;Sitka&quot;;*/
font: 18pt /*&quot;Segoe UI&quot;*/&quot;Simplex_IV25&quot;;
font-weight: 600;</string>
   </property>
   <property name="text">
    <string>РЕКОМЕНДОВАННЫЙ ВЕС</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Simplex_IV25</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 244, 243);
	color: #f00;
	border: 2px solid rgb(255, 82, 85);
    border-radius: 20px;
	font-family: /*&quot;Montserrat&quot;,Verdana,sans-serif*/&quot;Simplex_IV25&quot;;
	font-weight: 650;
}</string>
   </property>
   <property name="text">
    <string>Итоговый расчет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>500</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Simplex_IV25</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
	color: #f00;
	border: 2px solid rgb(255, 255, 255);
    border-radius: 20px;
	font-family: /*&quot;Montserrat&quot;,Verdana,sans-serif*/&quot;Simplex_IV25&quot;;
	font-weight: 600;
}</string>
   </property>
   <property name="text">
    <string>&lt;-назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>37</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Simplex_IV25</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font-family: /*&quot;Montserrat&quot;,Verdana,sans-serif*/&quot;Simplex_IV25&quot;;
font-weight: 550;</string>
   </property>
   <property name="text">
    <string>СМ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font-family: &quot;Montserrat&quot;,Verdana,sans-serif;
font-weight: 600;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>323</width>
     <height>179</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>80</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 255, 255);
/*color: rgb(69, 232, 24);
color: rgb(255, 173, 115);
/*font: 70pt &quot;Sitka&quot;;*/
font: 80pt &quot;Segoe UI&quot;;
/*font: 59pt &quot;Simplex_IV25&quot;;
font: 59pt &quot;SimSun&quot;;
font: 49pt &quot;Segoe UI Semibold&quot;;
font: 60pt &quot;Swis721 BlkCn BT&quot;;
font: 49pt &quot;Sitka Text&quot;;*/
color: rgb(11, 255, 202);
</string>
      </property>
      <property name="text">
       <string>60,0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Simplex_IV25</family>
        <pointsize>35</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
/*font-family: &quot;Montserrat&quot;,Verdana,sans-serif;*/
font: 35pt /*&quot;Sitka&quot;*/&quot;Simplex_IV25&quot;;
font-weight: 500;</string>
      </property>
      <property name="text">
       <string>кг</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Favicon.qrc"/>
  <include location="footer_img.qrc"/>
 </resources>
 <connections/>
</ui>
